docker

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-[ -f /etc/containers/nodocker ] || \
+[[ -f /etc/containers/nodocker || -d /etc/containers/nodocker ]] || \
```

Comment:
Why not just change the -f to -e, then if there is any file system object called /etc/containers/nodocker it will not show the message.

Reply:
Good call, I've made the change.